packages/create-plugin/templates/panel/README.md

```diff
@@ -10,8 +10,8 @@ Use panel plugins when you want to do things like visualize data returned by dat
 
 ## Getting started
 
--- INSERT FRONTEND GETTING STARTED --
 {{#if hasBackend}}-- INSERT BACKEND GETTING STARTED --{{/if}}
+-- INSERT FRONTEND GETTING STARTED --
```

Comment:
Hmm... we cannot have a backend for a panel plugin. Could we remove the `{{#if hasBackend}}-- INSERT BACKEND GETTING STARTED --{{/if}}` from this file instead?

Reply:
Yepp, I think it would be great to remove it! 👍